verification/curator-service/ui/src/components/App.test.tsx

```diff
@@ -41,31 +41,32 @@ it('renders without crashing when logged out', async () => {
         headers: {},
     };
     mockedAxios.get.mockResolvedValue(axiosResponse);
-    render(
+    const { queryByTestId } = render(
         <MemoryRouter>
             <App />
         </MemoryRouter>,
     );
     expect(mockedAxios.get).toHaveBeenCalledTimes(1);
     expect(mockedAxios.get).toHaveBeenCalledWith('/auth/profile');
+    expect(queryByTestId('profile-menu')).not.toBeInTheDocument();
 });
 
 it('has link to map', async () => {
     const axiosResponse = {
-        data: {},
-        status: 403,
-        statusText: 'Forbidden',
+        data: { name: 'Alice Smith', email: 'foo@bar.com', roles: ['admin'] },
```

Comment:
why do you need to be logged in to get access to the map?

Reply:
This was specifically checking for this button:
![Screen Shot 2020-09-23 at 9 10 04 AM](https://user-images.githubusercontent.com/30459667/93985138-98d3a280-fd7c-11ea-9112-502c00780303.png)

Which is only visible after logging in. There is another link to the map present on the landing page which is covered in the landing page test.